src/ORM/DataObjectSchema.php

```diff
@@ -673,19 +736,16 @@ protected function buildSortDatabaseIndexes($class)
      *
      * @param string $column String to parse containing the column name
      *
-     * @return array Resolved table and column.
+     * @return array Resolved table, column, and direction.
```

Comment:
I think it's extremely unlikely anyone would have been iterating over the result, it's intended to be used with `list($table, $column)` which will still work.